tad/api/routes/projects.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, Request
+from fastapi.responses import HTMLResponse
+
+from tad.api.deps import templates
+from tad.core.config import get_settings
+from tad.schema.instrument import InstrumentBase
+from tad.schema.project import ProjectNew
+from tad.schema.system_card import SystemCard
+from tad.services.instruments import InstrumentsService
+from tad.services.projects import ProjectsService
+from tad.services.storage import WriterFactory
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.get("/new")
+async def get_new(
+    request: Request,
+    instrument_service: Annotated[
+        InstrumentsService, Depends(InstrumentsService)
+    ],  # todo: create async_instruments_service
+) -> HTMLResponse:
+    instruments = instrument_service.fetch_instruments()
+
+    return templates.TemplateResponse(request, "projects/new.html.j2", {"instruments": instruments})
+
+
+@router.post("/new", response_class=HTMLResponse)
+async def post_new(
+    request: Request,
+    project_new: ProjectNew,
+    projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
+) -> HTMLResponse:
+    project = projects_service.create(project_new.name)
```

Comment:
nit-picking: I think (all) the steps below could be part of the project_service create method, as that would be a more logic place for 'business logic' steps than in the api call.

Reply:
I knew you where gona comment on this one ;)

Reply:
Moved into service